addons/editor-dark-mode/addon.json

```diff
@@ -1002,6 +1002,34 @@
         "darkScrollbars": true
       }
     },
+    {
+      "name": "Scratch 2.0 Style",
+      "id": "scratch2",
+      "description": "A theme based on Scratch 2.0's colors.",
+      "values": {
+        "page": "#ffffffff",
+        "primary": "#179fd7ff",
+        "highlightText": "#4d97ffff",
+        "menuBar": "#9c9ea2ff",
+        "activeTab": "#fafafaff",
```

Comment:
I think white or `#e6e8e8` (top or bottom of 2.0's gradient) would be more appropriate.